packages/client/src/forms/configuration/default/birth.ts

```diff
@@ -989,7 +989,46 @@ export const birthRegisterForms: ISerializedForm = {
             }
           ]
         }
-      ]
+      ],
+      mapping: {
+        template: [
+          {
+            fieldName: 'birthConfigurableIdentifier1',
+            operation: 'childIdentityToFieldTransformer',
+            parameters: [[IdentityIdType.BirthConfigurableIdentifier_1]]
+          },
+          {
+            fieldName: 'birthConfigurableIdentifier2',
+            operation: 'childIdentityToFieldTransformer',
+            parameters: [[IdentityIdType.BirthConfigurableIdentifier_2]]
+          },
+          {
+            fieldName: 'birthConfigurableIdentifier3',
+            operation: 'childIdentityToFieldTransformer',
+            parameters: [[IdentityIdType.BirthConfigurableIdentifier_3]]
+          }
+        ],
+        mutation: {
+          operation: 'childFieldToIdentityTransformer',
+          parameters: [
+            [
+              IdentityIdType.BirthConfigurableIdentifier_1,
+              IdentityIdType.BirthConfigurableIdentifier_2,
+              IdentityIdType.BirthConfigurableIdentifier_3
+            ]
+          ]
+        },
+        query: {
+          operation: 'childIdentityToFieldTransformer',
+          parameters: [
+            [
+              IdentityIdType.BirthConfigurableIdentifier_1,
+              IdentityIdType.BirthConfigurableIdentifier_2,
+              IdentityIdType.BirthConfigurableIdentifier_3
+            ]
+          ]
+        }
+      }
```

Comment:
@Zangetsu101 now that these are in Farajaland, can you please move them to the birth form there?  https://github.com/opencrvs/opencrvs-farajaland/pull/575